test/factories.rb

```diff
@@ -78,9 +78,9 @@
     after(:build) do |assn|
       assn.course = assn.teamset.course
     end
-    after(:create) do |assn, evaluator|
-      assn.graders << create(:grader, assignment: assn)
-    end
+    #after(:create) do |assn, evaluator|
```

Comment:
I think this breaks an invariant that assignments must have > 0 graders; this shouldn't be commented out.